vhdbuilder/scripts/windows/collect-windows-logs.ps1

```diff
@@ -96,6 +96,15 @@ else {
   Write-Host "Containerd hyperv logs not avalaible"
 }
 
+# log containerd containers (this is done for docker via networking collectlogs.ps1)
```

Comment:
add some logs for debugging?
Write-Host "Collecting Containerd logs"
Write-Host "Cannot find ctr.exe"
Write-Host "Collecting Containerd logs - containers"
Write-Host "Collecting Containerd logs - tasks"

Reply:
more logs are always good.